exchanges/bybit/bybit_wrapper.go

```diff
@@ -4,7 +4,6 @@ import (
 	"context"
```

Comment:
You've added kline.SevenHours, but it does not appear as a supported Feature under `Intervals: kline.DeployExchangeIntervals`